hle-survey/transformation/src/test/scala/org/broadinstitute/monster/dap/HealthTransformationsSpec.scala

```diff
@@ -42,13 +43,122 @@ class HealthTransformationsSpec extends AnyFlatSpec with Matchers {
     "hs_dx_infect_other_fu" -> Array("1")
   )
 
-  it should "correctly map infectious disease values when values are defined for a single disease" in {
-    val exampleInfectiousDiseaseRecord = RawRecord(id = 1, singleInfectiousDiseaseFields)
+  private val singleEyeDisease = Map[String, Array[String]](
```

Comment:
It looks like these input values are each only used in one test... would it make sense to move them into (or nearer to) the tests where they're used to make it easier to compare the input and output? I'm mostly suggesting that because this file will get pretty big as we add more health conditions.

Reply:
This is a good point! I have another PR coming in right after this; for the sake of getting this one through, I will move the input values around in the next PR.